src/interfaces/coral_web/src/components/Conversation/MessagingContainer.tsx

```diff
@@ -166,36 +151,26 @@ const Content: React.FC<Props> = (props) => {
   );
 };
 
-type MessagesProps = Props & { welcomeMessageEnabled: boolean };
+type MessagesProps = Props & { startOptionsEnabled: boolean };
 /**
  * This component is in charge of rendering the messages.
  */
 const Messages = forwardRef<HTMLDivElement, MessagesProps>(function MessagesInternal(
-  { welcomeMessageEnabled, onRetry, messages, streamingMessage, startOption, onStartOptionChange },
+  { startOptionsEnabled, onRetry, messages, streamingMessage, onPromptSelected },
   ref
 ) {
-  const lastMessage = messages[messages.length - 1];
-
+  const isConversationEmpty = messages.length === 0 && !streamingMessage;
```

Comment:
I think we can ignore `!streamingMessage` since the user message will always be added to `messages` first, did you find you needed this while testing?
```suggestion
  const isConversationEmpty = messages.length === 0;
```